vm/vm/src/tevm.spec.ts

```diff
@@ -305,6 +305,51 @@ describe('Tevm should create a local vm in JavaScript', () => {
 			expect(res.id).toBe(1)
 			expect(res.method).toBe('tevm_putContractCode')
 		})
+
+		it('Should fail a evm call request if the file is too large', async () => {
+			const tevm = await createTevm()
+			const address1 = '0x1f420000000000000000000000000000000000ff'
+
+			/*
+				Simple bytecode to deploy a large contract
+				`62` PUSH4, `FFFFFF` value being pushed, `60` PUSH1, `00` value, `F3` return
+				https://github.com/ethereumjs/ethereumjs-monorepo/blob/a0ef459e26f6a843d67bb2142977b67359109839/packages/evm/test/runCall.spec.ts#L543
+			*/
+			const data = '0x62FFFFFF6000F3'
+
+			const res = await tevm.request({
+				id: 1,
+				method: 'tevm_call',
+				jsonrpc: '2.0',
+				params: {
+					caller: address1,
+					data
+				},
+			})
+			expect(res.result.execResult.exceptionError?.error.toString()).toBe("code size to deposit exceeds maximum code size")
+		})
+
+		it('Should deploy large files if allowUnlimitedContractSize option is true', async () => {
+			const tevm = await createTevm({
+				allowUnlimitedContractSize: true
+			})
+			const address1 = '0x1f420000000000000000000000000000000000ff'
+
+			const data = '0x62FFFFFF6000F3'
+			const res = await tevm.request({
+				id: 1,
+				method: 'tevm_call',
+				jsonrpc: '2.0',
+				params: {
+					caller: address1,
+					data,
```

Comment:
I think the type for `RunCallAction` might be wrong. It requires a `to` address and `to` should be optional. Tsc is erroring here

Reply:
[RunCallAction has a to field and it is optional](https://github.com/evmts/tevm-monorepo/pull/769/files#diff-d13f73193588cb05003628a2cc8228929127379b9f13e0912167aab08eb35b34L7-R7)